README.rst

```diff
@@ -194,11 +200,11 @@ Usage
 		
```

Comment:
shouldn't the "- Open VirtualBox GUI " and the next bullet point be indented at the same level as "- Stop Docker Machine if..."?

Reply:
the "Restart Docker Machine, by running command :" also seems wrongly indented